src/packages/cli/src/types.ts

```diff
@@ -21,6 +21,7 @@ export type StartArgs<TFlavorName extends FlavorName> =
 export type GanacheArgs =
   | (AbstractArgs<"stop"> & { name: string })
   | AbstractArgs<"list">
+  | AbstractArgs<"none">
```

Comment:
This is required because Yargs parser no longer terminates the process when argument validation fails (or `--help`) is passed.